modules/web-console/web-agent/src/test/java/org/apache/ignite/console/agent/rest/RestExecutorSelfTest.java

```diff
@@ -504,6 +507,8 @@ public void testLongRunningQueries() throws Throwable {
      * @param qry Query text.
      */
     private boolean executeQuery(RestExecutor exec, String qry) throws Exception {
+        System.out.println(System.currentTimeMillis() + " " + qry + " start");
```

Comment:
Is that a debug code?

Reply:
Yes, I'm in progress of debug this test on TC. It is passing locally and failing on TC